heartbeat/mdraid

```diff
@@ -163,6 +165,37 @@ resource_is_cloned() {
 	[ -z "$OCF_RESKEY_CRM_meta_clone_max" ] && return 1 || return 0;
 }
 
+list_conf_arrays() {
+	test -f $mdadm_conf || {
+		ocf_exit_reason "$mdadm_conf gone missing!"
+		return $OCF_ERR_CONFIGURED
+	}
+	grep ^ARRAY $mdadm_conf | awk '{print $2}'
+}
+
+forall() {
+	local func=$1
+	local mddev rc=0
+
+	for mddev in $RAIDDEVS; do
+		# prepare md_dev and devs
+		md_dev="$mddev"
+
+		# Required by start|stop|monitor
+		devs="`list_devices_for_mddev`"
+		if [ `echo $devs|wc -l` -eq 0 ]; then
+			ocf_exit_reason "No component device(s) found for MD RAID array $md_dev"
+			rc=$(($rc | $OCF_ERR_CONFIGURED))
```

Comment:
This should be `$OCF_ERR_CONFIGURED`, so Pacemaker can react accordingly.

Reply:
Got it. As OCF_ERR_CONFIGURED is a fatal error, do we need just return here? Or let the loop continue?

Reply:
Okay. Just pushed with the update.